src/cityreader/cityreader.py

```diff
@@ -20,13 +38,20 @@ def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+  with open('cities.csv', newline='') as csvfile:
+    reader = csv.DictReader(csvfile)
+    for row in reader:
+      cities.append(City(name=row['city'], lat=row['lat'], lon=row['lng']))
```

Comment:
the values for lat and lng that your passing into the instance of City class needs to be passed as floats.  I believe the values are being passed as strings otherwise.